runelite-client/src/main/java/net/runelite/client/plugins/reportbutton/ReportButtonPlugin.java

```diff
@@ -71,6 +71,22 @@ ReportButtonConfig provideConfig(ConfigManager configManager)
 		return configManager.getConfig(ReportButtonConfig.class);
 	}
 
+	@Override
+	public void startUp()
+	{
+		updateReportButtonTime();
```

Comment:
Both this and shutdown are on the swing thread, not game thread. Also generally don't call `@Schedule`ed methods but instead make a separate method and call that.